INSTALL

```diff
@@ -101,7 +101,7 @@ Install required packages:
 
 Install optional packages:
 
-  vcpkg install qt5
+  vcpkg install qt5 luajit
```

Comment:
vcpkg installs lua & luajit into the same place, at least the version I have.

Could you add a sentence that if a user wants luajit then s/he might need to remove lua?

Reply:
@375gnu I think you are using an old vcpkg, it has been fixed in https://github.com/microsoft/vcpkg/pull/5863/commits 

Right now I am using 2019.07 tag version. It is no need to remove lua in order to install luajit.

Reply:
great